test/views/MoneyViewSpec.scala

```diff
@@ -48,13 +49,15 @@ class MoneyViewSpec extends ViewSpec with Mappings {
     val invalidSingleMoneyQuestion = formPageViewModelGen(singleQuestionGen(invalidMoneyForm))
     val viewModelGen = Gen.oneOf(singleMoneyQuestion, tripleMoneyQuestion, invalidSingleMoneyQuestion)
 
-    implicit val request = FakeRequest()
+    implicit val request: FakeRequest[AnyContentAsEmpty.type] = FakeRequest()
 
     "MoneyView" - {
 
       act.like(renderTitle(viewModelGen)(view(moneyForm, _), _.title.key))
       act.like(renderHeading(viewModelGen)(view(moneyForm, _), _.heading))
-      act.like(renderInputWithLabel(singleMoneyQuestion)("value", view(moneyForm, _), _.heading))
+      act.like(renderInputWithLabel(singleMoneyQuestion)("value", view(moneyForm, _), _.page.firstField.label))
+      // TODO: remove these tests? None of the pages that use MoneyView have multiple questions, so this seems redundant
+      //  Instead, create MultipleQuestionsViewSpec, to test pages with multiple money inputs?
 //      act.like(renderInputWithLabel(tripleMoneyQuestion)("value.1", view(_), _.page.fields.head.label))
 //      act.like(renderInputWithLabel(tripleMoneyQuestion)("value.2", view(_), _.page.fields(1).label))
 //      act.like(renderInputWithLabel(tripleMoneyQuestion)("value.3", view(_), _.page.fields(2).label))
```

Comment:
In addition to what the TODO says, the same is true for for IntView. I think we should create a MoneyViewSpec to include these kinds of tests, though I'm not sure if that work should be done here or in a separate ticket.